docs/features/jsonSchema.md

```diff
@@ -0,0 +1,17 @@
+The JSON Schema is structured to validate the yaml files that are loaded via JCasC.The structure and validation of the Schema is done based on the user-installed plugins.
+
+=== Version
+
+The Schema uses JSON draft v07. 
+http://json-schema.org/draft-07/schema#
+
+
+=== Progress
+
+* The JSON Schema is partially working and is in beta mode.
```

Comment:
Add more details about the current schema

Reply:
Definitely.